bot/src/bin/shuttle.rs

```diff
@@ -0,0 +1,16 @@
+#[shuttle_runtime::main]
+async fn shuttle_main() -> Result<MyService, shuttle_runtime::Error> {
+    Ok(MyService {})
+}
+
+// Customize this struct with things from `shuttle_main` needed in `bind`,
+// such as secrets or database connections
+struct MyService {}
+
+#[shuttle_runtime::async_trait]
+impl shuttle_runtime::Service for MyService {
+    async fn bind(self, _addr: std::net::SocketAddr) -> Result<(), shuttle_runtime::Error> {
+        // Start your service and bind to the socket address
+        Ok(())
+    }
+}
```

Comment:
:eyes:

Reply:
This sadly won't work as we need sync env with our own runtime...